apps/admin/src/components/common/SideBar.tsx

```diff
@@ -0,0 +1,58 @@
+import type { SidebarConfig } from "@/types/router";
+import { Button } from "@passu/ui/button";
+import { Divider } from "@passu/ui/divider";
+import { PassuLogo } from "@passu/ui/passu-logo";
+
+interface SidebarProps {
+  config: SidebarConfig;
+}
+
+export default function SideBar({ config }: SidebarProps) {
```

Comment:
```suggestion
export default function Sidebar({ config }: SidebarProps) {
```

nit; 단순 이름이긴 한데 불편하네요 ㅎㅎ;;

Reply:
[a40fd98](https://github.com/ssu-student-union/PASSUv2_FE/pull/12/commits/a40fd982d6beb6155e50e7d77049df1f1eaa4e20)
수정했습니다!